lib/cc/cli/prepare/quality.rb

```diff
@@ -0,0 +1,59 @@
+require "safe_yaml/load"
+
+module CC
+  module CLI
+    class Prepare
+      class Quality < Command
+        ENGINES_CONFIG = {
+          "complexity-ruby" => {
+            "enabled" => true,
+            "channel" => "beta",
+          },
+          "duplication" => {
+            "enabled" => true,
+            "channel" => "cronopio",
+          },
+        }.freeze
+
+        def execute
+          Dir.chdir(CC::Analyzer::MountedPath.code.container_path) do
+            content =
+              if (existing_contents = read_codeclimate_yml)
+                modify(existing_contents)
+              else
+                { "engines" => ENGINES_CONFIG }
+              end
+
+            write_codeclimate_yml(content)
+          end
+        end
+
+        private
+
+        def modify(content)
+          content.delete("ratings")
+          content["engines"] ||= {}
+          content["engines"].merge!(ENGINES_CONFIG)
+          content
+        end
+
+        def read_codeclimate_yml
+          SafeYAML.load_file(CODECLIMATE_YAML)
```

Comment:
Yeah, in this case I only need to know that it's valid as YAML to work with it in the way I need to, all the cc-yaml validation machinery will still happen downstream (as desired).

As this feature grows, we may start to move some functionality around between here and cc-yaml, but I'm deferring on that for now.